packages/block-editor/src/components/block-navigation/index.js

```diff
@@ -1,80 +1,28 @@
-/**
- * External dependencies
- */
-import { isArray, noop } from 'lodash';
-
 /**
  * WordPress dependencies
  */
-import { useDispatch, useSelect } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import BlockNavigationTree from './tree';
-import { isClientIdSelected } from './utils';
-import { store as blockEditorStore } from '../../store';
 
 export default function BlockNavigation( {
-	onSelect = noop,
+	onSelect = () => {},
```

Comment:
I think generally we should avoid this. Each time the block is rendered, it will get a new function reference. Thus rerendering the `BlockNavigationTree` below. We should either define a `const noop = () => {};` on the top, or use the one from `lodash`. I'd prefer the former.

Reply:
> Thus rerendering the `BlockNavigationTree` below. 

Won't this happen anyways since the tree is not a pure component?

Reply:
Oh that's right. Anyway, fewer props changing is always better 😄

Reply:
Agreed!